docs/devices/SNZB-05P.md

```diff
@@ -1,76 +0,0 @@
----
-title: "SONOFF SNZB-05P control via MQTT"
-description: "Integrate your SONOFF SNZB-05P via Zigbee2MQTT with whatever smart home infrastructure you are using without the vendor's bridge or gateway."
```

Comment:
why was this file modified?

Reply:
No, only added QA devices images.

Reply:
Thats not the case, look at the changes in the pr